src/redux/reducers/walletReducer.ts

```diff
@@ -99,23 +100,23 @@ function walletReducer(state = initialState, action: ActionType): WalletState {
   }
 }
 
-export function outpointToString(outpoint: Outpoint): string {
-  return `${outpoint.txid}:${outpoint.vout}`;
+export function toOutpoint(output: Outpoint): string {
```

Comment:
nit: I would rename toStringOutpoint (could be confusing with Outpoint from LDK?)